Src/FluentAssertions.DataSets/DataTableAssertionExtensions.cs

```diff
@@ -1,7 +1,9 @@
 ﻿using System.Data;
 using System.Diagnostics;
+using FluentAssertions.DataSets;
 using JetBrains.Annotations;
 
+// ReSharper disable once CheckNamespace
 namespace FluentAssertions;
```

Comment:
I love this, so users can use this library without importing an extra namespace.